tutorials/introductory/sample_plots.py

```diff
@@ -414,6 +414,7 @@
    :scale: 50
 
    xkcd
+
```

Comment:
Fixes

> /home/circleci/project/doc/tutorials/introductory/sample_plots.rst:425: WARNING: Explicit markup ends without a blank line; unexpected unindent.

which was caused by the original code translating to .rst as:

~~~
.. figure:: ../../gallery/showcase/images/sphx_glr_xkcd_001.png
   :target: ../../gallery/showcase/xkcd.html
   :align: center
   :scale: 50

   xkcd
Subplot example
===============
~~~